sample_apps/run_example.sh

```diff
@@ -2022,6 +2023,7 @@ function run_app_by_name_as_client_talk_to_Cert_Service() {
 
         # In order to get access to SWIG-generated *.py modules
         PYTHONPATH=../..; export PYTHONPATH
+        PYTHONUNBUFFERED=TRUE; export PYTHONUNBUFFERED
```

Comment:
... So that output from client- and server-portion of the app do not come out in interleaved order.